dla_cnn/data_model/DesiMock.py

```diff
@@ -0,0 +1,88 @@
+from astropy.io import fits
+import numpy as np
+
+
+class DesiMock:
+    """
+    a class to load all spectrum from a mock DESI data v9 fits file, each file contains 1186 spectra.
+    :attribute wavelength array-like, the wavelength of all spectrum (all spectrum share same wavelength array)
+    :attribute data, dict, using each spectra's id as its key and a dict of all data we need of this spectra as its value
+      its format like spectra_id: {'FLUX':flux,'ERROR':error,'z_qso':z_qso, 'RA':ra, 'DEC':dec, 'DLAS':dlas information} ,
+      dla information is a tuple, and its format is like (dla_id, z_qso, NHI)
+    :attribute split_point_br, the length of the flux_b,the split point of  b channel data and r channel data

```

Comment:
Please indicate this is an `int`

Reply:
Sorry for not checking in time, I have updated it.